.github/workflows/package.yaml

```
@@ -53,14 +53,39 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Download wheels for Python3.7
-        if: ${{ '3.7' == matrix.python-version }}
         uses: actions/download-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)